exporter.cfg

```diff
@@ -55,14 +55,14 @@ QueryJson = {
           },
           {
             "match_phrase": {
-              "payload.eventType.keyword": "JOB_TERMINAL"
+              "eventType.keyword": "JOB_TERMINAL"
             }
           },
           {
-            "range": {
+            "range": { 
               "@timestamp": {
-                "gte": "2020-07-18T22:10:43.388Z",
-                "lte": "2020-08-17T22:10:43.388Z",
+                "gte": "now-7d/d",
```

Comment:
we made it be dynamically ranged date starting from a week ago to now as an example. 
/d: Round down to the nearest day. reference: https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#date-math

Reply:
Yes, update it to be 2 minutes dynamic time range